### PR TITLE
Setup.py

### PR DESCRIPTION
**Pull request by UltralegendSam-Fs**
Setup.py

### CODE REVIEW

**Comment by UltralegendSam-Fs** [comment]
Modified Code & Update logo Design 
After update wifiphisher logo more attractive

**Comment by UltralegendSam-Fs** [comment]
But line 95 not changed by me

**Comment by pavonis9** [comment]
Yes I'm aware of that. You have added a space in the start of **line 96** and now **line 94** and **95** needs a space too in the start, after that you should remove one extra space in the last of the line. 

So the result should be like this : 
```
 print("                      _  __ _       _     _     _               ") 
 print("                     (_)/ _(_)     | |   (_)   | |              ") 
 print("   ((.))    __      ___| |_ _ _ __ | |__  _ ___| |__   ___ _ __ ") 
 print(r"    |      \ \ /\ / / |  _| | '_ \| '_ \| / __| '_ \ / _ \ '__|") 
 print(r"   /_\      \ V  V /| | | | | |_) | | | | \__ \ | | |  __/ |   ") 
 print(r"  /___\      \_/\_/ |_|_| |_| .__/|_| |_|_|___/_| |_|\___|_|   ") 
 print(r" /     \                    | |                                ") 
 print("                             |_|                                ") 
 print("                                                                ") 
 print("................................................................") 
```

instead of : 
```
 print("                     _  __ _       _     _     _                ") 
 print("                    (_)/ _(_)     | |   (_)   | |               ") 
 print("   ((.))    __      ___| |_ _ _ __ | |__  _ ___| |__   ___ _ __ ") 
 print(r"    |      \ \ /\ / / |  _| | '_ \| '_ \| / __| '_ \ / _ \ '__|") 
 print(r"   /_\      \ V  V /| | | | | |_) | | | | \__ \ | | |  __/ |   ") 
 print(r"  /___\      \_/\_/ |_|_| |_| .__/|_| |_|_|___/_| |_|\___|_|   ") 
 print(r" /     \                    | |                                ") 
 print("                             |_|                                ") 
 print("                                                                ") 
 print("................................................................") 
```

**Comment by UltralegendSam-Fs** [comment]
okk i

File: setup.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def get_dnsmasq():
 
 # setup settings
 NAME = "wifiphisher"
-AUTHOR = "sophron"
+AUTHOR = "Mr.Sophron"
```
Comment (sophron) [inline setup.py]: I'm having a hard time taking this PR seriously.

**Pull request closed without merging**